app/serializers/framework_framework_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class FrameworkFrameworkSerializer
+    include FastApplicationSerializer
+  
+    attributes :framework_id, :other_framework_id
+  
+    set_type :framework_framework
```

Comment:
@nrankin I wonder if this should be `set_type :framework_frameworks` (the latter as plural)?

Would be consistent with https://github.com/impactoss/impactoss-server/blob/master/app/serializers/recommendation_category_serializer.rb#L6 and https://github.com/impactoss/impactoss-server/blob/master/app/serializers/measure_indicator_serializer.rb#L6

Reply:
👍